### PR TITLE
add javascript filter box to script website

### PR DESCRIPTION
**Pull request by ailin-nemui**
and fix some html closing tags.

### CODE REVIEW

**Comment by dequis** [comment]
lol travis fails because cap_sasl.pl is blacklisted

File: index.html [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
         <hr>
 
+        <input type="search" class="light-table-filter" data-table="sortable" placeholder="Filter" size="100">
```
Comment (dequis) [inline index.html]: Use `style="width: 100%"` instead of size="100", which just means 100 characters

Comment (dequis) [inline index.html]: Oh also `class="light-table-filter form-control"` so that it uses the bootstrap css.

**Comment by dequis** [comment]
Just tested this, it's nice.

#188 addresses the travis issue.

**Comment by ailin-nemui** [comment]
thanks for the comments, implemented now

**Pull request merged**